internal/util/util.go

```diff
@@ -113,3 +113,9 @@ func AppendProjectEnvToCmdEnv(envMap map[string]string, envList []string) []stri
 	}
 	return envList
 }
+
+// TODO: indent each line for the modules.
+func IndentString(content string, spaces int) string {
```

Comment:
You could use a package like sprig for this which has easy to use support for templates. https://github.com/Masterminds/sprig

Reply:
I continued with a simple util function for indenting because I was not sure if sprig loops through each line to prefix each line with the right indentation.

Reply:
It does: http://masterminds.github.io/sprig/strings.html

Reply:
Fine to leave your function in there though, as it's already done.